cmake/evercbor.cmake

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache 2.0 License.
+
+# Build EverCBOR
+set(EVERCBOR_DIR "${CMAKE_CURRENT_SOURCE_DIR}/3rdparty/exported/evercbor")
```

Comment:
We have historically started with everything exported, in a quasi header-only situation, and have slowly pulled back in. Since this is a new dependency, I would like to try and start internal (as in 3rdparty/internal) and not export or install the headers. This probably requires some fixing on our side, so it is fine to start that way though.